specification/columns/capacityreservationstatus.md

```diff
@@ -0,0 +1,38 @@
+# Capacity Reservation Status
+
+Capacity Reservation Status indicates whether the charge represents either the consumption of the [*capacity reservation*](#glossary:capacity-reservation) identified in the CapacityReservationId column or when a *capacity reservation* is unused.
```

Comment:
```suggestion
Capacity Reservation Status indicates whether the charge represents the consumption of the [*capacity reservation*](#glossary:capacity-reservation) identified in the [CapacityReservationId](#capacityreservationid) column or the unused portion of that *capacity reservation*.
```

Reply:
This change seems unnecessary; is the current wording confusing?

Reply:
I'm okay with current - Irena can you explain why you want this?

Reply:
"_...when **a capacity reservation** is unused..._" caught my eye because we are not referring to the status of just any capacity reservation but specifically the one listed in the CapacityReservationId column. Therefore, we should **switch to "the"** .

I feel like the second version reads better, but if you prefer the first one, that's no problem."